integration/okhttp/src/main/java/com/bumptech/glide/integration/okhttp/OkHttpStreamFetcher.java

```diff
@@ -89,11 +89,13 @@ public void cancel() {
     // TODO: call cancel on the client when this method is called on a background thread. See #257
   }
 
+  @android.support.annotation.NonNull
```

Comment:
@sjudd Actually this is strange that the compilation and test passed, the refactoring was made by AS.
Integration libraries does not import Support Annotation libraries.
To import the NonNull I need to add the dependency is it ok for you or drop the annotation in those ?